valusets/ValueSet-England-DMMedicationRequestCourseOfTherapy.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-DMMedicationRequestCourseOfTherapy" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-DMMedicationRequestCourseOfTherapy" />
+    <version value="1.0.0" />
+    <name value="EnglandDMMedicationRequestCourseOfTherapy" />
+    <title value="England DMMedicationRequestCourseOfTherapy" />
```

Comment:
DMMedicationRequestCourseOfTherapy -> DM MedicationRequest Course Of Therapy

Reply:
Done with the required changes.